apps/studio/components/layouts/ProjectLayout/LayoutHeader/FeedbackDropdown/FeedbackWidget.tsx

```diff
@@ -201,29 +279,11 @@ export const FeedbackWidget = ({ onClose }: FeedbackWidgetProps) => {
             </motion.div>
           )}
         </AnimatePresence>
-      </div>
+      </>
 
       <PopoverSeparator_Shadcn_ />
 
-      <div className="px-5 flex flex-row justify-between items-start mt-4">
-        <div>
-          <p className="text-xs text-foreground">Have a technical issue?</p>
-          <p className="text-xs text-foreground-light">
-            Contact{' '}
-            <SupportLink>
-              <span className="cursor-pointer text-brand-link transition-colors hover:text-brand-600">
-                support
-              </span>
-            </SupportLink>{' '}
-            or{' '}
-            <a href={`${DOCS_URL}`} target="_blank" rel="noreferrer">
-              <span className="cursor-pointer text-brand-link transition-colors hover:text-brand-600">
-                see docs
-              </span>
-            </a>
-            .
-          </p>
-        </div>
```

Comment:
We don’t want to link straight to support ticket creation. Our now more “snappy” admonition is a better fallback in the case that the user writes support-type language.